src/app/k-tools/k-tools.component.ts

```diff
@@ -1,28 +1,79 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { Subscription } from 'rxjs';
 import { KToolsService } from './k-tools.service';
 import { KTool } from '../shared/k-tool.model';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
```

Comment:
move this under line1 with an empty line between core imports and app imports

Reply:
This okay?
<core imports>

<3rd party imports>

<app imports>

Reply:
Looks like it didnt save my whole comment. Core imports > break > 3rd party imports > break > app imports